website/js/lib/store/api/actions/testall.js

```diff
@@ -26,10 +26,12 @@ module.exports={
             url:context.rootState.info._links.jobs.href,
             method:'get'
         })
+        let body = opts.filter ? {filter:`${opts.filter}.*`} : {}
+        body.token = `${opts.token}`
```

Comment:
Could you please provide steps to tests this fix (test case with required assets (e.g., sample inputs)), so our QA can added to our test cases for the coming release.

Reply:
Sure.. 
Set QnABot setting "ENFORCE_VERIFIED_IDENTITY' to 'true'
Run TESTALL 
Without this fix, TESTALL results would all show 'Failed' with the bot response indicating that there is no verified identity.
With this fix, TESTALL results will show correct bot responses.